conans/client/graph/compute_pid.py

```diff
@@ -81,5 +82,10 @@ def run_validate_package_id(conanfile):
         with conanfile_exception_formatter(conanfile, "package_id"):
             with conanfile_remove_attr(conanfile, ['cpp_info', 'settings', 'options'], "package_id"):
                 conanfile.package_id()
-
+    else:
+        _conanfile_package_id_default(conanfile)
     conanfile.info.validate()
+
+
+def _conanfile_package_id_default(conanfile):
```

Comment:
Leftover helper? The `configure` ones do not have it and it feels like this one shouldn't either

Reply:
Yeah, better align with the configure() approach

Reply:
indeed, I will remove it